rules/windows/defense_evasion_msxsl_network.toml

```diff
@@ -1,8 +1,8 @@
 [metadata]
 creation_date = "2020/03/18"
-integration = ["endpoint", "windows"]
+integration = ["endpoint", "windows", "sentinel_one_cloud_funnel"]
 maturity = "production"
-updated_date = "2025/03/20"
+updated_date = "2025/08/26"
 
 [rule]
```

Comment:
this should be deprecated as its a dup of https://github.com/elastic/detection-rules/blob/fbfc696a86d3ec826fbd13fb8957d294f155dc17/rules/windows/defense_evasion_network_connection_from_windows_binary.toml#L138 and compatibility added to `Unusual Network Activity from a Windows System Binary`

Reply:
I'll take a look at it as part of https://github.com/elastic/ia-trade-team/issues/689